packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -464,6 +464,7 @@
   "Apps_Marketplace_Uninstall_Subscribed_App_Prompt": "This app has an active subscription and uninstalling will not cancel it. If you'd like to do that, please modify your subscription before uninstalling.",
   "Apps_Permissions_Review_Modal_Title": "This app requires the following permissions",
   "Apps_Permissions_No_Permissions_Required": "The App does not require additional permissions",
+  "Apps_Permissions_cloud_workspace-token": "Interact with Cloud Services on behalf of this server",
```

Comment:
@geekgonecrazy not sure about the wording here... are we calling this the "Cloud Services"? 🤔

Reply:
I think this is accurate.

Reply:
Just wondering if people will know what we're referring to 😛